### ISSUE TITLE
Supported link styles - setting

### ISSUE THREAD

**Issue by jonathan-s** [renerocksai/sublime_zk#58]
Documentation[1] mentions that there are a bunch supported link styles. Though it seems like I can't find a setting for those link styles in case you wanted to change to a different link style. 

[1]: https://github.com/renerocksai/sublime_zk#supported-link-styles

**Comment by renerocksai** [comment]
Yeah, the documentation is a bit misleading there. _Supported_ in this context means, syntax colouring, ctrl+enter, etc work with such links.

When the plugin inserts links, it currently only supports:

* [[wiki style]] (default)
* [single bracket] links

I've grown to like `[[201803192331 These kinds of links]]`, so probably when I have time, I'll add proper support for all link styles.

**Comment by cscottmills** [comment]
Here's a fork that adds a `move_title_inside_brackets` option:
https://github.com/bemilton/sublime_zk
